src/quickpicks/schemaRegistries.ts

```diff
@@ -1,45 +1,45 @@
 import * as vscode from "vscode";
 import { IconNames } from "../constants";
-import { getEnvironments } from "../graphql/environments";
 import { Logger } from "../logging";
 import { CCloudEnvironment } from "../models/environment";
 import { CCloudSchemaRegistry, SchemaRegistry } from "../models/schemaRegistry";
 import { hasCCloudAuthSession } from "../sidecar/connections";
+import { CCloudResourcePreloader } from "../storage/ccloudPreloader";
 import { getResourceManager } from "../storage/resourceManager";
+import { getSchemasViewProvider } from "../viewProviders/schemas";
 
 const logger = new Logger("quickpicks.schemaRegistry");
 
-/** Progress wrapper for the Schema Registry quickpick to accomodate data-fetching time. */
```

Comment:
The loading time is mostly zero now, given the prefetching, so I've done away with the progress bar outer layer function.

Reply:
One thing to note with this progress wrapper is that it will keep the loading element on the Schemas view itself while the quickpick is open (somewhat indicating that it's waiting on the user input). If we get rid of this one, we probably also want to get rid of the Kafka cluster quickpick view-progress indicator as well.

Reply:
Gotcha, good point. Per slack, we'll reintroduce this after all our stacked branches are merged in.